### PR TITLE
Chore/updates

### PR DESCRIPTION
**Pull request by ColdFire87**
- update npm modules & node engine versions

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
   include:
     - stage: install
       name: "Install Dependencies"
-      script: yarn install --frozen-lockfile
```
Comment (mikesalmon-mysense) [inline .travis.yml:16]: @ColdFire87 - any reason not to also use "frozen-lockfile" here to ensure it's not out of sync?

**Comment by mikesalmon-mysense** [comment]
Hey @ColdFire87 , for this, I also noticed travis ci failed, but due to auth issue with `fundingcircle.jfrog.io` - was that intentional to update `yarn.lock` references?

**Review by mikesalmon-mysense — CHANGES_REQUESTED** [review]
Multiple references to `fundingcircle.jfrog.io` in `yarn.lock` file - was that intentional?

**Comment by ColdFire87** [comment]
Hmm... I think it's because I did it from my work laptop, and I have the npm registry setup to use `fundingcircle.jfrog.io`.
I will reinstall the deps from my personal laptop and push the change.

**Pull request closed without merging**